lib/logstash/inputs/dynamodb.rb

```diff
@@ -0,0 +1,358 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require "logstash/inputs/base"
+require "logstash/namespace"
+require "securerandom"
+require "thread"
+require "socket"
+require_relative "LogStashRecordProcessorFactory"
+require_relative "DynamoDBLogParser"
+
+#Java Dependencies: KCL, Streams Adapter, Log4j, Preview AWS SDK
+require 'java'
```

Comment:
As I said before is better when you can locate all this require_jars in a specialised module so all this behaviour gets DRY out.